lib/commands/new.js

```diff
@@ -25,7 +22,7 @@ var repositories = {
 module.exports = function(args, options, callback, ee) {
   var projectName, projectFolder, framework, template, messages, directory;
   var tasks = [
-    preflight, prompt, gitClone, folderSetup, npmInstall, bowerInstall
+    preflight, prompt, gitClone, folderSetup, npmInstall
```

Comment:
Unrelated to the PR title.

Reply:
bower was removed from deps, because it is no longer maintained.